code/datums/gamemode/role/syndicate.dm

```diff
@@ -136,6 +136,7 @@
 /datum/role/traitor/challenger
 	name = CHALLENGER
 	id = CHALLENGER
+	name_for_uplink = "Challenger"
```

Comment:
why not use defines for these?

Reply:
Would a define work since it is specifically for the jobs_excluded check for the Available_for_jobs uplink proc?

Reply:
done, compiled, and tested